.travis.yml

```diff
@@ -135,3 +135,12 @@ matrix:
     - <<: *mbed-tools-test
       name: "Test sockets example (K64F)"
       env: EXAMPLE_NAME=mbed-os-example-sockets TARGET_NAME=K64F CACHE_NAME=sockets-K64F
+
+    # Mbed tools command line option test
+    - <<: *mbed-tools-test
+      name: "Test init command"
+      env: NAME=test-setup CACHE_NAME=test-setup
```

Comment:
Please change name and cache name to sensible name.
Also please move `_clone_dependencies` call in the matrix script call, as the `test init command` don't need them

Reply:
NAME and CACHE_NAME are just straight up copies from your PR. What do you suggest we change them to?

Moving the `_clone_dependencies` makes sense. I spotted that when I was looking at the travis expanded config, we are cloning the repos when we don't need them.

Reply:
`test-cmd-init`